src/pages/api/rpc/handler.ts

```diff
@@ -0,0 +1,40 @@
+import { PageConfig } from "next";
+import { NextRequest } from "next/server";
+
+import { chainRecord } from "@/chains/chains";
+import { getWhitelabelEndpoint } from "@/config/endpoints";
+import { getPolkachuAuthHeader } from "@/utils/api";
+
+export const config: PageConfig = {
+  api: {
+    externalResolver: true,
+    bodyParser: false,
+  },
+  runtime: "edge",
+};
+
+export default async function handler(req: NextRequest) {
+  try {
+    const [chainID, ...args] = req.url.split("/api/rpc/").pop()!.split("/");
```

Comment:
This logic is basically identical to the logic in the other handler, with a couple of strings changed. 

Surely we can do this more cleanly

Reply:
can refactor tomorrow 🙏🏻

Reply:
https://github.com/skip-mev/ibc-dot-fun/pull/160 @bpiv400